world/mujoco_ros/include/mujoco_ros/simulator.hpp

```diff
@@ -235,6 +236,40 @@ class simulator_visualizer_t
         goal_geom->pos[2] = 0.0;
       }
 
+      if (trajectory_to_visualize.size() > 0)
```

Comment:
This should be a separate function to add lines. From this, plotting a tree should be easy although maybe too expensive...

Reply:
Could you check if the current commit works?